lib/widgets/profile.dart

```diff
@@ -0,0 +1,334 @@
+                Text(user.fullName,
+                  style: const TextStyle(fontWeight: FontWeight.bold)),
+                ProfileEmailField(user: user),
+                ProfileRoleField(role: user.role),
+                // TODO(#197) render user status
+                // Text("Active about XXX"), // TODO(#196) render active status
+                LocalTimeField(user: user),
+                ProfileDataTable(user: user),
+                const SizedBox(height: 16),
+                FilledButton.icon(
+                  onPressed: () => _createDmNavigation(context),
+                  icon: const Icon(Icons.email),
+                  label: const Text("Send direct message")),
+              ])))));
+  }
+}
+
+class ProfileEmailField extends StatelessWidget {
+  final User user;
+
+  const ProfileEmailField({super.key, required this.user});
+
+  @override
+  Widget build(BuildContext context) {
+    if (user.deliveryEmailStaleDoNotUse == null) {
+      return const SizedBox.shrink();
+    }
+    return Text(user.deliveryEmailStaleDoNotUse!);
+  }
+}
+
+class ProfileRoleField extends StatelessWidget {
+  final UserRole role;
+
+  const ProfileRoleField({super.key, required this.role});
+
+  @override
+  Widget build(BuildContext context) {
+    final roleLabel = switch (role) {
+      UserRole.owner => "Owner",
+      UserRole.administrator => "Administrator",
+      UserRole.moderator => "Moderator",
+      UserRole.member => "Member",
+      UserRole.guest => "Guest",
+      UserRole.unknown => "Unknown",
+    };
+    return Text(roleLabel);
+  }
+}
+
+class LocalTimeField extends StatelessWidget {
+  final User user;
+
+  const LocalTimeField({super.key, required this.user});
+
+  @override
+  Widget build(BuildContext context) {
+    if (user.isBot) {
+      return const SizedBox.shrink();
+    }
+    final userLocalNow = getNowInTimezone(user.timezone);
+    if (userLocalNow == null) {
+      return const SizedBox.shrink();
+    }
+    return Text("${DateFormat.jm().format(userLocalNow)} Local time",
+      style: const TextStyle(fontWeight: FontWeight.normal));
+  }
+}
+
+class ProfileDataTable extends StatelessWidget {
+  final User user;
+
+  const ProfileDataTable({super.key, required this.user});
+
+  @override
+  Widget build(BuildContext context) {
+    if (user.profileData == null) {
+      return const SizedBox.shrink();
+    }
+    final store = PerAccountStoreWidget.of(context);
+    final displayEntries = sortAndAnnotateUserFields(user.profileData!, store.customProfileFields);
+    if (displayEntries.isEmpty) {
+      return const SizedBox.shrink();
+    }
+    return DefaultTextStyle.merge(
+      style: const TextStyle(fontSize: 16, height: 1),
+      textAlign: TextAlign.left,
+      child: Column(
+        children: [
+          const SizedBox(height: 16),
+          Table(
+            border: TableBorder.all(style: BorderStyle.none),
+            columnWidths: const {
+              0: FixedColumnWidth(116),
```

Comment:
Where does 116 come from?

Reply:
This was supposed to be based on the label width from the mobile app which was 96px for the unpadded text. I've since moved the padding around and didn't readjust the formula here (because this column needs to contain padded text). I've replaced it with `96 + 8` to match the old width with the new padding values.